worlds/mlss/Regions.py

```diff
@@ -0,0 +1,126 @@
+import typing
+
+from worlds.AutoWorld import World
+from BaseClasses import Region, Entrance
+from .Locations import MLSSLocation, mainArea, chucklehuck, castleTown, startingFlag, chuckolatorFlag, piranhaFlag, \
+    kidnappedFlag, beanstarFlag, birdoFlag, surfable, hooniversity, gwarharEntrance, gwarharMain, \
+    fungitown, fungitownBeanstar, fungitownBirdo, teeheeValley, winkle, sewers, airport, \
+    bowsers, bowsersMini, jokesEntrance, jokesMain, theater, booStatue, oasis, postJokes, baseUltraRocks, event, coins
+from . import StateLogic
+
+
+def create_regions(world: "World", excluded: typing.List[str]):
```

Comment:
```suggestion
if typing.TYPE_CHECKING:
    from . import MLSSWorld


def create_regions(world: "MLSSWorld", excluded: typing.List[str]):
```
Doing it like this gets your typing to be correct. Would recommend changing instances of world: World and world: "World" to world: "MLSSWorld" in this and other files